src/app/applications/applications.directive.app.js

```diff
@@ -3,12 +3,23 @@
 
   angular
   .module('roboconf.applications')
-  .directive('rbcfSingleApp', rbcfSingleApp);
+  .directive('rbcfSingleApp', rbcfSingleApp)
+  .directive('customOnChange', customOnChange);
```

Comment:
All our directives should start with `rbcf`.
And every directive should be defined in its own file.
You could call it `rbcfIconManagement`.